### DISCUSSION TITLE
2023 NetBox Community Survey

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#12392] [Announcements]
I'm happy to announce that our annual community survey has officially kicked off! Each year, we reach out to the entire NetBox user base with a short survey to learn more about who they are, how they use NetBox, and what they would most like to see in future releases. This feedback is crucial to guiding NetBox's long-term growth and focus.

The survey is a single page of 16 questions, and shouldn't take more than a few minutes to complete. As always, the results will be published at the end of May. (You can find last year's results [here](https://github.com/netbox-community/netbox/discussions/9026).) Your participation is greatly appreciated!

<div align="center">

### :ballot_box_with_check: [2023 NetBox Community Survey](https://forms.gle/zUHrrPo7K34yKaqC9)

</div>

**Edit:** The survey has concluded. Thank you everyone who participated! We'll publish the results soon (hopefully next week).

**Comment by aopdal** [comment]
As a network consultant I help my customers get going with NetBox and automation more than internal use. The survey does not catch this in a good way.
